adoc-mode.el

```diff
@@ -123,6 +124,38 @@ need to call `adoc-calc' for the change to take effect."
                  function)
   :group 'adoc)
 
+(defcustom adoc-hide-markup nil
+  "Determines whether markup in the buffer will be hidden.
+When set to nil, all markup is displayed in the buffer as it
+appears in the file.  An exception is when `adoc-hide-urls'
+is non-nil.
+Set this to a non-nil value to turn this feature on by default.
+You can interactively toggle the value of this variable with
+`adoc-toggle-markup-hiding', \\[adoc-toggle-markup-hiding],
+or from the Markdown > Show & Hide menu.
+Markup hiding works by adding text properties to positions in the
+buffer---either the `invisible' property or the `display' property
+in cases where alternative glyphs are used (e.g., list bullets).
+This does not, however, affect printing or other output.
+Functions such as `htmlfontify-buffer' and `ps-print-buffer' will
+not honor these text properties.  For printing, it would be better
+to first convert to HTML or PDF (e.g,. using Pandoc)."
+  :group 'adoc
+  :type 'boolean
+  :safe 'booleanp
+  :package-version '(adoc-mode . "0.7.0"))
+(make-variable-buffer-local 'adoc-hide-markup)
+
+(defcustom adoc-uri-types
```

Comment:
Can't see other references to this new defcustom either.

Reply:
Removed.